src/test/java/com/patrycja/pound/resource/AnimalResourceTest.java

```diff
@@ -0,0 +1,67 @@
+package com.patrycja.pound.resource;
+
+import com.patrycja.pound.models.dto.AnimalDTO;
+import com.patrycja.pound.models.dto.CatDTO;
+import com.patrycja.pound.services.AnimalService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(MockitoJUnitRunner.class)
+@WebMvcTest
```

Comment:
Do you need it? Will it work with `MockitoJUnitRunner.class` or is it only for `SpringRunner.class`?